pkg/kubelet/status/status_manager.go

```diff
@@ -613,3 +613,78 @@ func normalizeStatus(pod *v1.Pod, status *v1.PodStatus) *v1.PodStatus {
 	kubetypes.SortInitContainerStatuses(pod, status.InitContainerStatuses)
 	return status
 }
+
+// mergePodStatus merges oldPodStatus and newPodStatus where pod conditions
```

Comment:
Is that because it's never called ??

Reply:
It's code that is present in the release Lyft applied their changes on, but that is not part of their change. I should have deleted it, and will do now.